internal/commands/permission.go

```diff
@@ -73,6 +74,7 @@ func RegisterPermissionCmd(rootCmd *cobra.Command) *cobra.Command {
 	_ = checkCmd.Flags().MarkHidden("revision")
 	checkCmd.Flags().Bool("explain", false, "requests debug information from SpiceDB and prints out a trace of the requests")
 	checkCmd.Flags().Bool("schema", false, "requests debug information from SpiceDB and prints out the schema used")
+	checkCmd.Flags().Bool("error-on-no-permission", false, "if true, zed will return exit code 1 if subject does not have unconditional permission")
```

Comment:
bike shedding but maybe `--exit-1-without-membership`? I don't really like either

Can we find prior art for a flag like this in other software?

Reply:
I did a cursory search but couldn't find anything before I came up with this name randomly. I'm fine with whatever